include/mbedtls/config.h

```diff
@@ -3262,6 +3262,20 @@
  */
 #define MBEDTLS_VERSION_C
 
+/**
+ * \def MBEDTLS_PKCS7_USE_C
```

Comment:
Minor: This should be `MBEDTLS_PKCS7_C` and ideally be placed between `MBEDTLS_PKCS5_C` and `MBEDTLS_PKCS11_C`.